Composer/packages/server/src/locales/en-US.json

```diff
@@ -1553,8 +1553,8 @@
   "get_started_with_bot_framework_composer_57a6d38b": {
     "message": "Get started with Bot Framework Composer"
   },
-  "get_started_with_your_new_bot_da9aaf6a": {
-    "message": "Get started with your new bot"
+  "get_your_bot_up_and_running_418dd8ea": {
```

Comment:
do you want this change to be a part of this PR?

Reply:
Yes, it's an intended change, this string was modified in some other PR but locale json was not updated